Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/changedetection/CombinedRiskLevelChangeDetector.kt

```diff
@@ -46,14 +43,17 @@ class CombinedRiskLevelChangeDetector @Inject constructor(
 
     fun launch() {
         Timber.v("Monitoring combined risk level changes.")
-        riskLevelStorage.latestCombinedEwPtRiskLevelResults
+        riskLevelStorage.allCombinedEwPtRiskLevelResults
             .map { results ->
```

Comment:
Need to use all results because they are being filtered now, to make sure there are still two result left to compare